cfbs/git.py

```diff
@@ -177,3 +177,25 @@ def git_discard_changes_in_file(file_name):
         raise CFBSGitError(
             "Failed to discard changes in file '%s'" % file_name
         ) from cpe
+
+
+def git_check_tracked_changes(scope=["all"]):
+    should_commit = False
+    try:
+        result = run(["git", "status", "-s", "-u"], check=True, stdout=PIPE)
+        if "all" in scope:
+            if len(result.stdout) > 0:
+                should_commit = True
+        else:
+            changes = list(
+                map(
+                    lambda m: m.strip().split(" ")[1],
+                    list(filter(None, result.stdout.decode("utf-8").split("\n"))),
+                )
+            )
+            should_commit = any(i in changes for i in scope)
+        if not should_commit:
+            print("No changes to commit")
+        return should_commit
+    except CalledProcessError as cpe:
+        return False
```

Comment:
When does this happen? Deserves at least a comment.

Reply:
It might happen if, for example the command fired not in a git repository
I'm not sure what should be the expected behavior :) 
Maybe it's better to raise Exception?

Reply:
Ah, I see.

In that case I'd probably do:

```
user_error("Failed to run 'git status -s -u' to check for changes.")
```

Also, please ensure that your change works for projects not using git committing (`"git": false` in `cfbs.json`).